File: Baekjoon/14913.py
a, d, k = map(int, input().split())

n = 1
while True:
    num = a + (n - 1) * d
    if num == k:
        print(n)
        break
    elif d > 0 and num > k:
        print("X")
        break
    elif d < 0 and num < k:
        print("X")
        break
    n += 1

File: Baekjoon/14913.kt
fun main() {
    val (a, d, k) = readLine()!!.split(" ").map { it.toInt() }
    var cur = 1
    while (true) {
        val num = a + (cur - 1) * d
        if (num == k) {
            println(cur)
            break
        } else if ((d > 0 && num > k) || (d < 0 && num < k)) {
            println("X")
            break
        }
        cur++
    }
}